lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1339,6 +1339,13 @@ enum AddressMaskRange {
   eAddressMaskRangeAll = eAddressMaskRangeAny,
 };
 
+/// Useful for callbacks whose return type indicates
+/// whether to continue iteration or short-circuit.
+enum class IterationMarker {
```

Comment:
Maybe `IterationAction` could be better? @jimingham @clayborg Any thoughts?

Reply:
IterationAction seems more descriptive